src/CILogViewer.php

```diff
@@ -6,58 +6,43 @@
  */
 namespace CILogViewer;
 
-defined('BASEPATH') OR exit('No direct script access allowed');
-defined('APPPATH') OR exit('Not a Code Igniter Environment');
-
-
 class CILogViewer {
 
-    private $CI;
-
     private static $levelsIcon = [
+        'CRITICAL' => 'glyphicon glyphicon-error-sign',
         'INFO'  => 'glyphicon glyphicon-info-sign',
         'ERROR' => 'glyphicon glyphicon-warning-sign',
         'DEBUG' => 'glyphicon glyphicon-exclamation-sign',
         'ALL'   => 'glyphicon glyphicon-minus',
     ];
 
     private static $levelClasses = [
+        'CRITICAL' => 'danger',
         'INFO'  => 'info',
         'ERROR' => 'danger',
         'DEBUG' => 'warning',
         'ALL'   => 'muted',
     ];
 
-
-    const LOG_LINE_START_PATTERN = "/((INFO)|(ERROR)|(DEBUG)|(ALL))[\s\-\d:\.\/]+(-->)/";
-    const LOG_DATE_PATTERN = ["/^((ERROR)|(INFO)|(DEBUG)|(ALL))\s\-\s/", "/\s(-->)/"];
-    const LOG_LEVEL_PATTERN = "/^((ERROR)|(INFO)|(DEBUG)|(ALL))/";
+    const LOG_LINE_HEADER_PATTERN = '/^([A-Z]+)\s*\-\s*([\-\d]+\s+[\:\d]+)\s*\-\->\s*(.+)$/';
 
     //this is the path (folder) on the system where the log files are stored
-    private $logFolderPath;
+    private $logFolderPath = WRITEPATH . 'logs/';
 
     //this is the pattern to pick all log files in the $logFilePath
-    private $logFilePattern;
+    private $logFilePattern = "log-*.log";
 
     //this is a combination of the LOG_FOLDER_PATH and LOG_FILE_PATTERN
     private $fullLogFilePath = "";
 
-    //these are the config keys expected in the config.php
-    const LOG_FILE_PATTERN_CONFIG_KEY = "clv_log_file_pattern";
-    const LOG_FOLDER_PATH_CONFIG_KEY = "clv_log_folder_path";
-
-
-
     /**
-     * Here we define the paths for the view file
-     * that's used by the library to present logs on the UI
+     * Name of the view to pass to the renderer
+     * Note that for it allows namespaced views if your view is outside
+     * the View folder.
+     *
+     * @var string
      */
-    private $LOG_VIEW_FILE_FOLDER = "";
-    private $LOG_VIEW_FILE_NAME = "logs.php";
-    private $LOG_VIEW_FILE_PATH = "";
-
-    //this is the name of the view file passed to CI load->view()
-    const CI_LOG_VIEW_FILE_PATH = "cilogviewer/logs";
+    private $viewName = "App\ThirdParty\CILogViewer\Views\logs";
```

Comment:
```suggestion
    private $viewName = "CILogViewer\Views\logs";
```

Using `App\ThirdParty\CILogViewer\Views\logs` will not work out of the box and will require package users to create/copy the view file in that directory